command/login.go

```diff
@@ -216,7 +216,7 @@ func (c *LoginCommand) Run(args []string) int {
 	}
 
 	// Create the client
-	client, err := c.Client()
+	client, err := c.ClientWithoutToken()
```

Comment:
In particular, down below where `token` is set, I'd add an else condition to re-trigger the helper logic on (new) lines 189...196 above in `command/base.go`. 

Other than that, LGTM!